Checking for adapter's native createParamAccessor

ahwayakchih opened the pull request:
This change makes node-any-db-params check for adapter's native `createParamAccessor()`, as discussed on https://github.com/grncdr/node-any-db-adapter-spec/pull/7#issuecomment-57315898. If there is any, it will be used instead of this modules' implementation.

ahwayakchih committed:
Checking for adapter's native createParamAccessor

This change makes node-any-db-params check for adapter's native `createParamAccessor()`, as discussed on https://github.com/grncdr/node-any-db-adapter-spec/pull/7#issuecomment-57315898. If there is any, it will be used instead of this modules' implementation.

grncdr commented on index.js:
```diff
@@ -16,7 +16,7 @@ function createParameterContext (queryable) {
     throw new TypeError('Argument must be a valid Queryable');
   }
-  var impl = exports.adapters[anyDbAdapter.name];
+  var impl = adapter.createParamAccessor || exports.adapters[anyDbAdapter.name];
```
the `adapter` variable isn't defined

ahwayakchih replied:
You're right, sorry for that. Should work better with next commit :).

ahwayakchih committed:
Fixed my stupid mistake

As mentioned here: https://github.com/grncdr/node-any-db-params/pull/1/files#r20759873

grncdr commented:
Thanks, sorry for taking so long to get back to this. It's been released as 1.0.1

ahwayakchih commented:
Thanks.
